dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -76,8 +76,8 @@ BEGIN
     AUTOCHECKBOX "Micr&osoft CHAP versiunea 2 (MS-CHAP v2)", 1492, 28, 176, 200, 10, BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 28, 189, 208, 1
     AUTOCHECKBOX "Pentru protocoalele &bazate pe MS-CHAP, utilizează nume și parole de autentificare în ReactOS (și eventual, domenii)", 1549, 28, 195, 213, 24, BS_TOP | BS_MULTILINE
-    DEFPUSHBUTTON "Con&firmă", 1, 127, 232, 60, 14
-    PUSHBUTTON "A&nulează", 2, 192, 232, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 127, 232, 60, 14
```

Comment:
This change introduces an accelerator collision with the line
 AUTOCHECKBOX "Micr&osoft CHAP versiunea 2 (MS-CHAP v2)", 1492, 28, 176, 200, 10, BS_TOP | BS_MULTILINE
Please check your romanian Windows to find out which of both lines need to be changed.